addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -299,14 +299,6 @@
             parent="menu_livechat_root"
             sequence="55"/>
 
-        <menuitem
-            id="canned_responses"
-            name="Canned Responses"
-            parent="livechat_config"
-            action="mail.mail_shortcode_action"
-            groups="im_livechat_group_user"
-            sequence="15"/>
```

Comment:
Don't we want to keep a link to the config from the livechat menu for ease of access?

Reply:
It doesn't hurt I guess